opentracing-spring-cloud-starter-jaeger/README.md

```diff
@@ -0,0 +1,113 @@
+## Configuration
+
+```xml
+<dependency>
+  <groupId>io.opentracing.contrib</groupId>
+  <artifactId>opentracing-spring-cloud-starter-jaeger</artifactId>
+</dependency>
```

Comment:
Could you please add directly here a description of the default behaviour. .e.g how it sends data to Jaeger and how to override it. 

This section should provide minimal configuration to make it work.

Reply:
Ideally the section should provide a shortcut how to set UDP or HTTP sender